.pre-commit-config.yaml

```diff
@@ -41,3 +41,5 @@ repos:
     rev: v3.3.1
     hooks:
       - id: pyupgrade
+        args:
+          - "--keep-runtime-typing"
```

Comment:
What does it?

Reply:
Without it, pyupgrade changes Optional/Union statements by new syntax "a | b" (Python v3.10+), which won't be completely backwards compatible.